### PR TITLE
Improve the CLI for debug mode

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

**Comment by sblaisot** [comment]
😻

File: cmd/instances.go [hunk 1]
```diff
@@ -355,27 +355,54 @@ instance of the given domain. Set the quota to 0 to remove the quota.
 }
 
 var debugInstanceCmd = &cobra.Command{
-	Use:   "debug <domain> <true/false>",
+	Use:   "debug <true/false>",
```
Comment (sblaisot) [inline cmd/instances.go]: Dors it break current behaviour ? Maybe you need to advertise this to the team ?

Comment (nono) [reply cmd/instances.go]: No, it doesn't break. I have kept compatibility with the old way: it detects the number of arguments to know if the domain is a positional argument or a flag.

Comment (sblaisot) [reply cmd/instances.go]: :+1:

Comment (sblaisot) [reply cmd/instances.go]: :+1:

**Pull request merged**